forest/daemon/src/daemon.rs

```diff
@@ -149,13 +148,8 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
 
     {
         // Start Prometheus server port
-        let prometheus_listener =
-            TcpListener::bind(config.client.metrics_address).unwrap_or_else(|_| {
-                cli_error_and_die(
-                    format!("could not bind to {}", config.client.metrics_address),
-                    1,
-                )
-            });
+        let prometheus_listener = TcpListener::bind(config.client.metrics_address)
+            .context("could not bind to metrics address")?;
```

Comment:
Looks like you're changing the error message. Could you show what the error message looks like before and after this change?

Reply:
Reverted to what it was